GitCommands/RevisionReader.cs

```diff
@@ -351,13 +344,12 @@ DateTime ParseUnixDateTime()
 
                     if (c == '\n')
                     {
-                        return DateTimeUtils.UnixEpoch.AddTicks(unixTime * TimeSpan.TicksPerSecond).ToLocalTime();
+                        return unixTime;
                     }
 
                     unixTime = (unixTime * 10) + (c - '0');
                 }
             }
-
             #endregion
```

Comment:
Nit: restore empty line before `#endregion`

Reply:
checked the file